messages/1.1.68.md

```diff
@@ -0,0 +1,24 @@
+=> 1.1.68
+
+## Breaking Changes
```

Comment:
Since you made it work with the old setting, it's no longer a breaking change and I would either remove this update message or at least change this heading to `Deprecations` or such.

Reply:
`Deprecations` notice will be preferred :)
[`58cceab` (#70)](https://github.com/sublimelsp/LSP-pyright/pull/70/commits/58cceabd6b1b1f9c2e2803c4750c8adb76fab5e7)